libmamba/src/core/query.cpp

```diff
@@ -599,7 +592,7 @@ namespace mamba
         }
 
         std::vector<mamba::printers::FormattedString> headers;
-        std::vector<std::string_view> cmds, args;
+        std::vector<std::string> cmds, args;
```

Comment:
This is the bug as `cmds` and `args` were refering to [sfmt](https://github.com/mamba-org/mamba/blob/main/libmamba/src/core/query.cpp#L627-L633) after it was out of scope, and this was only appearing in osx and windows...